warehouse/transformer/logger.go

```diff
@@ -72,6 +73,12 @@ func (t *Transformer) sampleDiff(eventsToTransform []types.TransformerEvent, pRe
 		if len(diff) == 0 {
 			continue
 		}
+
+		// JS converts new Date('0001-01-01 00:00').toISOString() to 2001-01-01T00:00:00.000Z
+		// https://www.programiz.com/online-compiler/4SqZcIH5k6Yli
+		if strings.Contains(diff, "\"0001-01-01T00:00:00.000Z\"") {
```

Comment:
It is possible that `diff` contains both `0001-01-01T00:00:00.000Z` and some other diff. In that case we will miss it, rt?

Reply:
It's likely but that should be fine.